site/docs/components/pill/accessibility.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Pill
+layout: DetailComponent
+sidebar:
+  label: Accessibility
+data:
+  $ref: ./#/data
+---
+
+### Keyboard interactions
+
+<KeyboardControls>
+  <KeyboardControl keyOrCombos="Enter or Space">
+    - When focus is on Pill, it is activated.
+  </KeyboardControl>
+  <KeyboardControl keyOrCombos="Tab">
+    - Moves focus into Pill. Focus is set to the entire Pill. - When Pill has
```

Comment:
this second bullet point should be on the next line
if it's prettier, might need to ignore